spec/models/provider_spec.rb

```diff
@@ -212,4 +213,69 @@
       it { is_expected.to be false }
     end
   end
+
+  describe "#performance_sign_offs" do
+    context "when there are no sign offs" do
+      let(:provider) { create(:provider) }
+
+      it "returns an empty collection" do
+        expect(provider.performance_sign_offs).to be_empty
+      end
+    end
+
+    context "when there are only census sign offs" do
+      let(:census_sign_off) { create(:sign_off, :census) }
+      let(:provider) { create(:provider, sign_offs: [census_sign_off]) }
```

Comment:
Can we work the factory
```suggestion
      let(:provider) { create(:provider, :census_sign_off) }
```